### PR TITLE
RATS: Remove Docker from integration tests

### PR DESCRIPTION
**Pull request by jakelandis**
This is in prep for a Docker based test run approach, and by removing the dependent Docker containers we will avoid Docker in Docker requirements.

This is actually @andrewvc 's work, I am just porting it from a feature branch. 

Lets wait for Travis and Jenkins to pass before merging to ensure I actually ported it correctly.

### CODE REVIEW

**Comment by original-brownbear** [comment]
@jakelandis need a review here? (just asking because it goes to the feature branch, happy to help if yes :))

**Comment by jakelandis** [comment]
@original-brownbear - yes please.  I pulled this out of the feature branch to its own PR  since the change stands alone and the existing Docker based tests was giving @untergeek some trouble.

**Comment by original-brownbear** [comment]
@jakelandis alrighty then, on it :)

File: qa/integration/services/kafka_teardown.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+stop_kafka() {
+    echo "Stopping Kafka broker"
+    $KAFKA_HOME/bin/kafka-server-stop.sh
+    echo "Stopping zookeeper"
+    $KAFKA_HOME/bin/zookeeper-server-stop.sh
+}
+
+# delete test topic
+echo "Deleting test topic in Kafka"
+$KAFKA_HOME/bin/kafka-topics.sh --delete --topic logstash_topic_plain --zookeeper localhost:2181 --if-exists
```
Comment (original-brownbear) [inline qa/integration/services/kafka_teardown.sh]: @jakelandis this is probably not necessary if we delete the kafka dir afterwards? (just trying to save some Travis minutes :D )

**Comment by original-brownbear** [comment]
@jakelandis LGTM :)

**Comment by jakelandis** [comment]
will need a manual back port to 5.5 when the builds change.

**Pull request closed without merging**